pandas/tests/indexing/test_chaining_and_caching.py

```diff
@@ -365,12 +365,6 @@ def check(result, expected):
     def test_cache_updating(self):
         # GH 4939, make sure to update the cache on setitem
 
-        df = tm.makeDataFrame()
-        df["A"]  # cache series
-        df.ix["Hello Friend"] = df.ix[0]
-        assert "Hello Friend" in df["A"].index
-        assert "Hello Friend" in df["B"].index
```

Comment:
Should this be kept but with something else as ix? 

Also, need to remove the filterwarnings mark I think